library/src/main/kotlin/com/connectrpc/protocols/ErrorJSONModels.kt

```diff
@@ -36,3 +36,7 @@ internal class EndStreamResponseJSON(
     @Json(name = "error") val error: ErrorPayloadJSON?,
     @Json(name = "metadata") val metadata: Headers?,
 )
+
+internal fun contentTypeIsJSON(contentType: String): Boolean {
+    return contentType == "application/json" || contentType == "application/json; charset=utf-8"
```

Comment:
Should we use `MediaType` to parse these reliably (we use this in `ConnectOkHttpClient.kt`)?